test/VoucherHub.test.ts

```diff
@@ -996,10 +996,29 @@ describe('VoucherHub', function () {
         });
     });
 
+    describe('Is voucher', function () {
```

Comment:
Let's add a test where a voucher contract is deployed manually not through the VoucherHub and check that `isVoucher`, and `getVoucher` return respectively `false` and `0x0`.

Reply:
As discussed, a voucher contract deployed manually won't be able to land on a legitimate voucher address